commerce/endpoint/account/address/classes/UsZipCleansingSample.cls

```diff
@@ -0,0 +1,32 @@
+public with sharing class UsZipCleansingSample {
+
+    private static Pattern validZipPattern = Pattern.compile('(^\\d{5}$)|(^\\d{9}$)|(^\\d{5}-\\d{4}$)');
```

Comment:
NIT: I'm curious whether apex/SF has standard patterns predefined that we could use. Some patterns, such as email address and phone number, can be a nightmare to self define and are often wrong. If we do define these, it may be worth showing a best practice of reusing those patterns.

Reply:
Didn't see anything.